methcomp/blandaltman.py

```diff
@@ -11,33 +12,18 @@ class _BlandAltman(object):
     """Internal class for drawing a Bland-Altman plot"""
 
     def __init__(self, method1, method2,
-                 x_title, y_title, graph_title,
-                 diff, limit_of_agreement, reference, CI,
-                 xlim, ylim,
-                 color_mean, color_loa, color_points,
-                 point_kws):
+                 diff, limit_of_agreement, CI):
```

Comment:
Use type annotations here:
`diff: str, limit_of_agreement: float, CI: float`

Reply:
changed
thanks, I will now leave the method1 and method2 documentation for what it is - as comparator will fix this

Reply:
now I realize that these arguments are already provided in the method (e.g. blandaltman()), should we also provide them double here?

Reply:
While I also don't love duplication, I think it is warranted here. It makes using the code much easier.

Reply:
I will keep it in.